src/main/java/com/cloudbees/hudson/plugins/folder/computed/FolderComputation.java

```diff
@@ -122,7 +123,7 @@
     private volatile Result result;
 
     /** The past few durations for purposes of estimation. */
-    @CheckForNull
+    @Nullable // until set
```

Comment:
Not sure why SpotBugs did not previously complain about https://github.com/jenkinsci/cloudbees-folder-plugin/blob/945ab2ef03dba53db977e7a47badd6d7c458224d/src/main/java/com/cloudbees/hudson/plugins/folder/computed/FolderComputation.java#L183-L189

Reply:
I do not see why this is not just initialised to an empty list here and made final simplifying the code elsewhere.
the null check here is only to prevent creation - and that will cause it to be set whenerver there is a run, and elsewhere the code is guarded by both the null check *and* an emptyness check.

Are we likely to have a Computation that will never run at least once?

Reply:
> why this is not just initialised to an empty list here and made final

Because it would still be null in previously serialized computations.

Reply:
🤦 for some reason I did not think this was serialized